### PR TITLE
fix: new error handling for predictions and tests

### PR DESCRIPTION
**Pull request by atierian**
## Swift SDK Update 0.26.0

Makes necessary changes to get AWSPredictionsPlugin and CoreMLPredictionsPlugin to compile.

## Debt Introduced
- There's a new error branch in `AWSPredictionsService+Polly` that now throws an `.unknown` error. Take another look at it to see if there's a more appropriate error than can be thrown there.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: AmplifyPlugins/Predictions/AWSPredictionsPlugin/Service/Predictions/AWSPredictionsService+Polly.swift [hunk 1]
```diff
@@ -34,12 +34,18 @@ extension AWSPredictionsService: AWSPollyServiceBehavior {
...
-        } catch let error as SynthesizeSpeechOutputError {
-            throw ServiceErrorMapping.synthesizeSpeech.map(error)
+            switch speech {
+            case .data(let data?):
+                let textToSpeechResult = Predictions.Convert.TextToSpeech.Result(
+                    audioData: data
+                )
+                return textToSpeechResult
+                // TODO: figure out what to throw here
+            default: throw PredictionsError.unknown("Missing respose", "", nil)
```
Comment (atierian) [inline AmplifyPlugins/Predictions/AWSPredictionsPlugin/Service/Predictions/AWSPredictionsService+Polly.swift:44]: Added a note to this and the base PR to come back to this.

File: AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Rekognition+PredictionsErrorConvertible.swift [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import AWSRekognition
+import Amplify
+import ClientRuntime
+
+
+extension AWSRekognition.HumanLoopQuotaExceededException: PredictionsErrorConvertible {
+    var predictionsError: PredictionsError {
+        .service(
+            .init(
+                description: "",
```
Comment (phantumcode) [inline AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Rekognition+PredictionsErrorConvertible.swift:17]: is this intended to have no description or recovery suggestion?

File: AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Textract+PredictionsErrorConvertible.swift [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+import AWSTextract
+import Amplify
+import ClientRuntime
+
+
+extension AWSTextract.HumanLoopQuotaExceededException: PredictionsErrorConvertible {
+    var predictionsError: PredictionsError {
+        .service(
+            .init(
+                description: "",
```
Comment (phantumcode) [inline AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Textract+PredictionsErrorConvertible.swift:17]: same question as previous comment, is the empty strings intended?

File: AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Rekognition+PredictionsErrorConvertible.swift (hunk 2 above)
Comment (atierian) [reply AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Rekognition+PredictionsErrorConvertible.swift]: Thanks for catching that. These were previously without description / recovery suggestion as well. Addressed in [f13a59a](https://github.com/aws-amplify/amplify-swift/pull/3257/commits/f13a59a4dd5e7767cc614347a072aaa41ec81089)

File: AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Textract+PredictionsErrorConvertible.swift (hunk 3 above)
Comment (atierian) [reply AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/ErrorHandling/Textract+PredictionsErrorConvertible.swift]: Addressed in [f13a59a](https://github.com/aws-amplify/amplify-swift/pull/3257/commits/f13a59a4dd5e7767cc614347a072aaa41ec81089)

**Review by phantumcode — APPROVED** [review]

**Pull request merged**